infra-templates/gandi-dns/0/rancher-compose.yml

```diff
@@ -0,0 +1,41 @@
+.catalog:
```

Comment:
Please add the `# notemplating` tag to the top of the file and `minimum_rancher_version: v1.6.0-rc1`. See https://github.com/rancher/community-catalog/blob/master/infra-templates/digitalocean-dns/3/rancher-compose.yml#L1